test/fixtures/courses.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
I think let's go ahead and remove the fixtures for now, and we can always add them back if we need them.